setup/kubernetes/index.md

```diff
@@ -1,13 +1,21 @@
 ---
 title: Kubernetes
-description: Learn how to set up a K8 cluster for your Coder deployment.
+description: Learn how to set up a Kubernetes cluster for your Coder deployment.
 ---
 
-You can install and deploy Coder onto any Kubernetes cluster (version 1.13.7+)
-that meets the [requirements](../requirements.md).
+You can deploy Coder into any [compatible Kubernetes cluster]. We recommend
+following the [Kubernetes upstream version support policy] in order to receive
```

Comment:
I wrote kind of a vague statement here instead of a specific version, but I think we should explicitly _require_ that people use a supported version of Kubernetes, so that we can rely on Kubernetes features available in 1.18 (currently the oldest version in support).

Reply:
Maybe we tie it in to K8 support -- if they do, we do (so we'd support the three minor releases K8 does); then we could write a generic statement and link to this [doc](https://kubernetes.io/docs/setup/release/version-skew-policy/)